app/models/hackney/keyfax_session.rb

```diff
@@ -0,0 +1,15 @@
+class Hackney::KeyfaxSession
+  include ActiveModel::Model
+
+  attr_accessor :launch_url, :guid
+
+  def self.get_startup_url(current_page_url)
```

Comment:
Can you please make this one `def self.create(current_page_url:)`? This way the model will quack like ActiveRecord.

Reply:
:sparkles: